### DISCUSSION TITLE
How to edit navigation bar

### DISCUSSION THREAD

**Discussion by juliusandretti** [MrGreensWorkshop/MrGreen-JekyllTheme#106] [How to] [answered]
Hi, I would like to know how to add new entries to the navigation bar. I'd like to have something similar to the navigation bar I've setup in my page (https://condor-instruments.github.io/actigraphy-tutorials-sample/) but as a side navigation bar, like the default one on the theme, but I don't know which files to edit. I'd also like to thank you for making this awesome theme available.

**Comment by MrGreensWorkshop** [comment, marked as answer]
You need to add new layout, inside _layout folder. Button name needs to be defined in _data/lang/[language].yml
for example check out about page /tabs/about.md and its layout _layouts/about.md
